lib/utils/transcription-utils.js

```diff
@@ -789,7 +789,7 @@ module.exports = (logger) => {
       };
     }
     else if ('deepgram' === vendor) {
-      let {model} = rOpts;
+      let model = rOpts.deepgramOptions.model || rOpts.model;
```

Comment:
I think needs to be `rOpts.deepgramOptions?.model` in case the user does not provide a deepgramOptions property?

Reply:
yes good catch, i didn't think about that scenario, tried other comibantion of model